software/ADPL_electron/ADPL_electron.ino

```diff
@@ -72,6 +78,12 @@ void setup() {
     attachInterrupt(UP, up_pushed, FALLING);
     attachInterrupt(DOWN, down_pushed, FALLING);
     attachInterrupt(RESET, res_pushed, FALLING);
+
+    if (SDCARD) {
+        pinMode(SD_CS_PIN, OUTPUT);
+        SD.begin(SD_CS_PIN);
+        sdFile = SD.open("adpl_data.txt", FILE_WRITE);  // FILE_WRITE should append existing file
```

Comment:
should we write a header or something indicating we are coming out of a reboot to the file? Since the timestamp resets I suppose we'll know that way anyway, but just a thought.